package.json

```diff
@@ -75,15 +76,33 @@
     "webpack-dev-server": "^3.11.0"
   },
   "devDependencies": {
+    "angular-mocks": "1.3.20",
+    "chai": "3.0.0",
+    "chai-datetime": "1.4.0",
+    "chai-shallow-deep-equal": "1.4.0",
     "css-loader": "^3.6.0",
     "exports-loader": "^1.0.0",
     "expose-loader": "^1.0.0",
     "file-loader": "^6.0.0",
     "glob-all": "^3.2.1",
+    "html-loader": "^1.1.0",
     "html-webpack-plugin": "^4.3.0",
+    "ignore-loader": "^0.1.2",
+    "imports-loader": "^1.1.0",
+    "karma": "4.2.0",
+    "karma-mocha": "1.1.1",
+    "karma-firefox-launcher": "1.1.0",
+    "karma-sinon-chai": "^2.0.2",
+    "karma-spec-reporter": "0.0.26",
+    "karma-webpack": "^4.0.2",
     "less-loader": "^6.1.1",
+    "mocha": "6.2.0",
+    "mockery": "1.7.0",
```

Comment:
Sorry I missed this but it is also not needed.

Reply:
aight, thanks